platform/broadcom/one-pde-image.mk

```diff
@@ -0,0 +1,81 @@
+# sonic broadcom one image installer
+
+SONIC_ONE_PDE_IMAGE = sonic-broadcom-pde.bin
+$(SONIC_ONE_PDE_IMAGE)_MACHINE = broadcom
+$(SONIC_ONE_PDE_IMAGE)_IMAGE_TYPE = onie
+$(SONIC_ONE_PDE_IMAGE)_INSTALLS += $(BRCM_OPENNSL_KERNEL)
+$(SONIC_ONE_PDE_IMAGE)_INSTALLS += $(PDDF_PLATFORM_MODULE)
+$(SONIC_ONE_PDE_IMAGE)_LAZY_INSTALLS += $(DELL_S6000_PLATFORM_MODULE) \
```

Comment:
>$(SONIC_ONE_PDE_IMAGE)_LAZY_INSTALLS [](http://example.com/codeflow?start=0&length=36)

This is duplicated as `$(SONIC_ONE_IMAGE)_LAZY_INSTALLS`, hard to maintain.

Reply:
Removed